src/connectors/scene/CryptoExchangeProcessingQuoteConnector.js

```diff
@@ -0,0 +1,23 @@
+// @flow
```

Comment:
This definitely doesn't need to be a connected component. You can just pass `shiftPendingTransaction` to this Component via the Action as a parameter. You'd save a bunch of code and remove one more mapStateToProps that would execute on every single dispatch.